tests/lax_scipy_test.py

```diff
@@ -132,7 +135,14 @@ def testScipySpecialFun(self, scipy_op, lax_op, rng_factory, shapes, dtypes,
     self._CompileAndCheck(lax_op, args_maker, rtol=1e-5)
 
     if test_autodiff:
-      jtu.check_grads(lax_op, args, order=1,
+      def partial_lax_op(*vals):
+        list_args = list(vals)
+        for i in nondiff_argnums:
```

Comment:
If I'm reading this correctly, this assumes `nondiff_argnums` is sorted and unique. it might be worth adding an explicit check that `list(nondiff_argnums) == sorted(set(nondiff_argnums))` before this.

Reply:
Good catch! I think we can do this sorting job at the definition of `op_record`. WDYT?

Reply:
Sure, that sounds like a good approach.